website/docs/containerization/05-overview.md

```diff
@@ -4,13 +4,14 @@ title: Container Overview
 ---
 ## Related Videos
 
-<VideoContainer
-  list={[{
-   src: "https://www.youtube-nocookie.com/embed/jq9wLQpDtvQ",
-   title: "Container Overview"
-  }]}
+<video
+  url={[
+    "https://www.youtube-nocookie.com/embed/jq9wLQpDtvQ",
+  ]}
```

Comment:
Not sure about this, the look and feel is different (e.g. no youtube player controls available)
What was the issue with `VideoContainer`?